[2024-08-30T08:49:20.713+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-08-30T08:49:20.733+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag_with_python_operator_v03.get_name manual__2024-08-30T08:49:19.813377+00:00 [queued]>
[2024-08-30T08:49:20.743+0000] {taskinstance.py:2603} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag_with_python_operator_v03.get_name manual__2024-08-30T08:49:19.813377+00:00 [queued]>
[2024-08-30T08:49:20.744+0000] {taskinstance.py:2856} INFO - Starting attempt 1 of 6
[2024-08-30T08:49:20.763+0000] {taskinstance.py:2879} INFO - Executing <Task(PythonOperator): get_name> on 2024-08-30 08:49:19.813377+00:00
[2024-08-30T08:49:20.772+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=3910) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-08-30T08:49:20.772+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_dag_with_python_operator_v03', 'get_name', 'manual__2024-08-30T08:49:19.813377+00:00', '--job-id', '12', '--raw', '--subdir', 'DAGS_FOLDER/python_operator_dag.py', '--cfg-path', '/tmp/tmpt2g2juup']
[2024-08-30T08:49:20.774+0000] {standard_task_runner.py:72} INFO - Started process 3911 to run task
[2024-08-30T08:49:20.774+0000] {standard_task_runner.py:105} INFO - Job 12: Subtask get_name
[2024-08-30T08:49:20.824+0000] {task_command.py:467} INFO - Running <TaskInstance: my_dag_with_python_operator_v03.get_name manual__2024-08-30T08:49:19.813377+00:00 [running]> on host efc48c282cbe
[2024-08-30T08:49:20.904+0000] {taskinstance.py:3122} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ubeyd' AIRFLOW_CTX_DAG_ID='my_dag_with_python_operator_v03' AIRFLOW_CTX_TASK_ID='get_name' AIRFLOW_CTX_EXECUTION_DATE='2024-08-30T08:49:19.813377+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-30T08:49:19.813377+00:00'
[2024-08-30T08:49:20.905+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-08-30T08:49:20.918+0000] {python.py:240} INFO - Done. Returned value was: Jerry
[2024-08-30T08:49:20.944+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-08-30T08:49:20.945+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=my_dag_with_python_operator_v03, task_id=get_name, run_id=manual__2024-08-30T08:49:19.813377+00:00, execution_date=20240830T084919, start_date=20240830T084920, end_date=20240830T084920
[2024-08-30T08:49:20.989+0000] {local_task_job_runner.py:261} INFO - Task exited with return code 0
[2024-08-30T08:49:21.005+0000] {taskinstance.py:3891} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-30T08:49:21.009+0000] {local_task_job_runner.py:240} INFO - ::endgroup::
